docs/ui/components/Layout/LayoutScroll.tsx

```diff
@@ -4,37 +4,49 @@ import React, { forwardRef, HTMLAttributes, PropsWithChildren } from 'react';
 
 type LayoutScrollProps = PropsWithChildren<
   HTMLAttributes<HTMLDivElement> & {
-    /** If the scroll container should smoothly scroll when scrolled programatically */
+    // If the scroll container should smoothly scroll when scrolled programatically
```

Comment:
Why did you change this from jsdoc to inline comment? I think it's part of the props, so jsdoc would be better here (bc vscode will just add this description to hovering over that prop when using the component)

Reply:
I thought that since this type is not exported the comments in there are just notes, not the actual docs. 😅 

I will change back the comment format.

Reply:
It's done.